app/views/layouts/application.html.erb

```diff
@@ -8,6 +8,7 @@
     <% end %>
     <%= csrf_meta_tags %>
     <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track': 'reload' %>
+    <link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/semantic-ui/2.2.12/semantic.min.css"></link>
```

Comment:
Is it good to have these external references? What if that page disappears?
cc @trevorb

Reply:
Yep, it's probably a better idea to include this in our standard css package rather than depend on an external cdn.